subsys/bluetooth/host/conn_internal.h

```diff
@@ -159,7 +159,7 @@ struct acl_data {
 
 struct bt_conn {
 	uint16_t			handle;
-	uint8_t			type;
+	enum bt_conn_type	type : 8;
```

Comment:
```suggestion
	enum bt_conn_type	type;
```

Even though this isn't a public header, my comment w.r.t. bitfields in header file still applies. 

Unless it is required to be specific size, suggest to just use the `enum` without the bitfield.